app/services/reactivate_account_session.rb

```diff
@@ -28,7 +28,10 @@ def suspend
   # @param [Pii::Attributes]
   def store_decrypted_pii(pii)
     reactivate_account_session[:personal_key] = true
-    reactivate_account_session[:pii] = pii.to_json
+    pii_json = pii.to_json
+    reactivate_account_session[:pii] = pii_json
+    Pii::Cacher.new(@user, session).save_decrypted_pii_json(pii_json)
```

Comment:
This will write the value in both places, and after the next deploy, we can remove `reactivate_account_session[:pii]` and change the `decrypted_pii` to read from `Pii::Cacher`

Reply:
One side effect of this is if the person is going through account reactivation during an IAL2 request, they will have the decrypted_pii in the expected spot, and after copying their new personal key and clicking "Continue to $SP", they won't have to enter their password again because the PII is already decrypted in the expected location.